content/getting-started/community.md

```diff
@@ -5,11 +5,30 @@ prev: "/getting-started/solutions/"
 
 # Community
 
-<!--
-TODO Kata discussion page, posts, spoilers, post labels
-TODO How to communicate effectively
-TODO Where to ask for help
-TODO Community rules
-TODO Link to Zulip when ready
-TODO Link to Forum when ready
--->
+Codewars gathers thousands of users of many skills and levels of experience. Their knowledge is invaluable resource and when used correctly, your training can be easier, and extremely effective.
```

Comment:
```suggestion
Codewars gathers thousands of users with many different skills and levels of experience. Their knowledge is an invaluable resource and they can make your training easier, and extremely effective.
```

As Dan said in gitter about the "trouble shooting" document, starting with something a bit negative (like "when used correctly") might not be welcoming (even if true). In addition, "using" people leaves me a bad feeling when I read it. ;o

Reply:
I don't get the part about "used correctly" being negative. Does it sound so?

About "using" part, it's about using the knowledge, and not about using people. It's quite common form in my language, does it indeed sound wrong in English?

Reply:
> Codewars gathers thousands of users

Not sure if we should mention the size of the community. But this is inaccurate since we have over 1.7M users total and at least 400K active users in the past 6 months.